src/common/components/nav/desktopNav.tpl.html

```diff
@@ -54,7 +54,7 @@
           <ul class="pull-right" uib-dropdown-menu>
             <span class="triangle"></span>
             <li class="nav-edit-profile"><a href="profile.html" translate>Edit Profile</a></li>
-            <li class="nav-sign-out"><a href="" ng-click="$ctrl.signOut()" translate>Sign Out</a></li>
+            <li class="nav-sign-out"><a href ng-click="$ctrl.signOut()" translate>Sign Out</a></li>
```

Comment:
I believe this should be `href=""`.  I remember issues with IE 9 and 10 not recognizing the anchor tag as a link without it.